app/src/main/java/com/example/activeamigo/CalendarActivity.java

```diff
@@ -12,12 +12,15 @@
 import android.widget.TextView;
 import android.widget.Toast;
 
+import androidx.appcompat.app.ActionBar;
 import androidx.appcompat.app.AppCompatActivity;
 
 public class CalendarActivity extends AppCompatActivity {
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
```

Comment:
/** Back button: function is not to save edited data to firebase **/
    @Override
    public boolean onOptionsItemSelected(@NonNull MenuItem item) {
        switch (item.getItemId()) {
            case android.R.id.home:
                this.finish();


                return true;
        }
        return super.onOptionsItemSelected(item);
    }
    
    try including this to end activity

Reply:
refer to https://github.com/CSE-210/Exercise-Buddy-App/blob/PreferencePage/app/src/main/java/com/example/activeamigo/PreferenceActivity.java#L187C1-L197C6